src/screens/regimen/SaloonVisitScreen/visits/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+
+import {
+    Container,
+    Content,
+    Time,
+    Title,
+    Purpose,
+    Booking,
+    Book
+} from './style'
+import { useThemeContext } from '../../../../theme';
```

Comment:
move it up

Reply:
sorry, I'm always forgetting some of them